.schema/openapi/patches/courier.yaml

```diff
@@ -0,0 +1,29 @@
+# Makes messageStatus a string enum
+- op: remove
+  path: /components/schemas/messageStatus/format
+- op: replace
+  path: /components/schemas/messageStatus/type
+  value: string
+- op: add
+  path: /components/schemas/messageStatus/enum
+  value:
+    - queued
+    - sent
+    - processing
+    - abandoned
+# Makes messageType a string enum
+- op: remove
+  path: /components/schemas/messageType/format
+- op: replace
+  path: /components/schemas/messageType/type
+  value: string
+- op: add
+  path: /components/schemas/messageType/enum
+  value:
+    - email
+    - phone
```

Comment:
Maybe make this just one op?
```suggestion
- op: replace
  path: /components/schemas/messageStatus
  value:
    type: string
    enum:
      - queued
      - sent
      - processing
      - abandoned
# Makes messageType a string enum
- op: replace
  path: /components/schemas/messageType
  value:
    type: string
    enum:
      - email
      - phone
```
That way it is exactly what you want.

Reply:
Yeah that's way better approach, idk what i was thinking writing that piece of code :-)